tests/Unit/Css/StyleRuleTest.php

```diff
@@ -65,7 +65,9 @@ public function hasAtLeastOneDeclarationForDeclarationBlockWithOneRuleReturnsTru
     }
 
     /**
-     * @return array<string, array{0: array<int, array{property: string, value: string}>, 1: string}>
+     * @return array<string, array{
+     *     0: array<int, array{property: non-empty-string, value: non-empty-string}>, 1: string
```

Comment:
With the array type definition now spanning multiple lines, I think it would be clearer if each array element was on a separate line, as we would do for PHP multiline array definitions.  Though I suspect a trailing comma on the last line wouldn't work here (have not tested), so the analogy ends there.

```suggestion
     *     0: array<int, array{property: non-empty-string, value: non-empty-string}>,
     *     1: string
```

Reply:
Don't commit this suggestion - see below.